bedrock/firefox/templates/firefox/products/desktop.html

```diff
@@ -12,6 +12,11 @@
 
 {% block body_id %}firefox-product-desktop{% endblock %}
 
+{% block optimizely %}
+  {% if switch('firefox-products-optimizely', ['en-US']) %}
```

Comment:
Is this only for the desktop page or should it appear on all product pages? If it's only desktop, should the switch name make that explicit? `firefox-product-desktop-optimizely` ?